lively.morphic/rendering/animations.js

```diff
@@ -109,6 +109,8 @@ export class PropertyAnimation {
   constructor (queue, morph, config) {
     this.queue = queue;
     this.morph = morph;
+    this.isStyleApplication = config.isStyleApplication;
+    delete config.isStyleApplication;
```

Comment:
Why do we need to delete this?

Reply:
Animations are messy and I wanted to avoid non morphing properties from entering the animations code. I will deal with the separately in a animations-refactoring issue. Maybe when we tackle the renderer altogether.